pmm-app/src/pmm-qan-app-panel/QueryAnalytics/StateContext.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useEffect, useState } from 'react';
+import { ParseQueryParamDate } from '../../react-plugins-deps/components/helpers/time-parameters-parser';
+import { getDataSourceSrv } from '@grafana/runtime';
+import _ from 'lodash';
```

Comment:
It's much better to import individual functions, rather than the whole big thing :)
`import { find } from 'lodash';`